README.md

```diff
@@ -69,6 +69,7 @@ The maintainers of RFCs will review the proposal, ask if there's any objections,
 | [13](rfcs/0013-get-block-template) | [Block Template](rfcs/0013-get-block-template/0013-get-block-template.md) | Dingwei Zhang | Standards Track | Proposal |
 | [14](rfcs/0014-vm-cycle-limits) | [VM Cycle Limits](rfcs/0014-vm-cycle-limits/0014-vm-cycle-limits.md) | Xuejie Xiao | Standards Track | Proposal |
 | [17](rfcs/0017-tx-valid-since) | [Transaction valid since](rfcs/0017-tx-valid-since/0017-tx-valid-since.md) | Jinyang Jiang | Standards Track | Proposal
+| [19](rfcs/0019-data-structures) | [Data Structures](rfcs/0019-data-structures/0019-data-structures.md) | Haichao Zhu, Xuejie Xiao | Standards Track | Proposal
```

Comment:
```suggestion
| [19](rfcs/0019-data-structures) | [Data Structures](rfcs/0019-data-structures/0019-data-structures.md) | Haichao Zhu, Xuejie Xiao | Informational | Draft
```

`Informational` is more appropriate as this is an RFC reflects design decisions formed somewhere else.